lesson-17/index.js

```diff
@@ -0,0 +1,200 @@
+// CW17
+
+// # Class
+// ## Task 1
+// Напишите функцию min(a,b), которая возвращает меньшее из чисел a и b.
+
+// function findMin(a,b) {
+//     return Math.min(a, b)
+// }
+
+// console.log(findMin(40,13));
+
+// ## Task 2
+// Напишите функцию pow(x,n), которая возвращает x в степени n. Иначе говоря, умножает x на себя n раз и возвращает результат.
+
+// P.S. В этой задаче функция обязана поддерживать только натуральные значения n, т.е. целые от 1 и выше.
+
+// function findPow(x,n) {
+//     if (!Number.isInteger(n)) {
+//         return 
+//     }
+//     else if (n > 0) {
+//         return Math.pow(x, n)
+//     }
+//     else {
+//         return;
+//     }
+// }
+// console.log(findPow(4, -1.5));
+
+// ## Task 3
+// Поиск гласных
+
+// Нужно написать функцию, принимающую строку в качестве аргумента и возвращающую количество гласных, которые содержатся в строке.
+// Гласными являются «a», «e», «i», «o», «u».
+// var s = prompt("enter smth", "")
+// function vowelsRed (s) {
+//    var vs="aeiouy";
+//    let sum = 0;
+//    for(let char of s) {
```

Comment:
........................
```suggestion
//    for (let char of s) {
```